README.md

```diff
@@ -26,9 +28,9 @@ The CSL specifies [three different contribution roles](https://github.com/finos/
 
 The following Working Groups are currently activated or about to be setup for this project:
 
-* [Contribution Review Working Group](https://www.finos.org/finos-community-calendar)
-* [Collateral Working Group](https://www.finos.org/finos-community-calendar)
-* [Technology Architecture Working Group](https://www.finos.org/finos-community-calendar)
+* [Contribution Review Working Group](https://github.com/eteridvalishvili/common-domain-model/files/11547016/CDM.Contribution.Review.Working.Group.Terms.of.Reference.pdf)
+* [Collateral Working Group](https://github.com/eteridvalishvili/common-domain-model/files/11547285/CDM-for-Collateral-Initiatives-factsheet.1.pdf)
+* [Technology Architecture Working Group](https://github.com/eteridvalishvili/common-domain-model/files/11546939/CDM.Technology.Architecture.WG.ToR.v1.2.4.pdf).
```

Comment:
These links reference your own GitHub fork. The files should be positioned in the main repo and the links updated

Reply:
Hi @lolabeis I think everything is ready for merging. I've replaced the links with the Terms of Reference PDFs which I added in the "docs" directory and updated the readme with relative links to ensure they work after merging, without requiring any additional changes.

As for the structure products WG, since it doesn't have one yet, I've left it linked to the community calendar for now.

Let me know if you have any questions.

Reply:
Hi @lolabeis Just wanted to follow up and see if this is okay to be merged.

Reply:
@eteridvalishvili - the issue is still there. you can simply search on `README.md` for `eteri` and you'll spot the issues mentioned by Leo.

CC @TheJuanAndOnly99

Reply:
Confirming that this is fixed now!